raiden/utils/cli.py

```diff
@@ -353,6 +381,7 @@ def apply_config_file(
         default_config_missing = (
             ex.errno == errno.ENOENT
             and config_file_path.resolve() == config_file_default_path.resolve()
+            and config_file_option_name in options_using_default
```

Comment:
Why is this necessary? From the name I'd expect that the comparison is `config_file_option_name not in options_using_default`